server/controllers/memory.js

```diff
@@ -0,0 +1,12 @@
+const assert = require('assert');
+const { mysql } = require('../qcloud.js');
+const trainingTableName = "Training";
+
+async function get(ctx) {
+  var memory = await mysql(trainingTableName).orderBy('UpdatedTime', 'desc').select('*').limit(10);
```

Comment:
Can we rename the variable and the file as "trainings"? It's not clear what "memory" means.